docs/source/index.rst

```diff
@@ -40,6 +40,7 @@ The `hi-ml` toolbox provides
    :caption: Histopathology
 
    histopathology.md
+   dsa.md
```

Comment:
if this is the title that appears in the hierarchy maybe I'd call it digital_slide_archive

Reply:
That's just the file name. The docs look like this: https://hi-ml--417.org.readthedocs.build/en/417/dsa.html